widgets/src/Project/Page.jsx

```diff
@@ -137,7 +152,7 @@ return (
       <HeaderDetails>
         <Widget
           src={`${ownerId}/widget/Project.HeaderDetails`}
-          props={{ accountId: props.accountId }}
+          props={{ accountId: ownerId, isAdmin }}
```

Comment:
Since we changed where the `isAdmin` status is stored (e.g. in `state`) we also need to pass it into props differently, like this:
```javascript
  props={{ accountId, isAdmin: state.isAdmin }}
```

Note that the `accountId` prop should also be the `accountId` variable, not the `ownerId` (this is due to hardcoding it to `ownerId` when doing mocks while developing the early version, so next time we should sink on this.